s3-cloudfront-spa/example-pattern.json

```diff
@@ -0,0 +1,63 @@
+{
+  "title": "AWS CloudFront and Amazon S3 for Single Page Application Hosting",
+  "description": "This project shows how to host a Single-Page Application (SPA) using AWS CloudFront & S3. CloudFront serves the SPA content from an S3 bucket configured for static hosting.",
```

Comment:
Please reword to stay below 150 characters

Reply:
updated to 149 characters